resources/lang/en/store.php

```diff
@@ -89,6 +89,7 @@
         'paid_on' => 'Order placed :date',
 
         'invoice' => 'View Invoice',
+        'no_orders' => 'No orders to view.',
```

Comment:
I don't think `to view` part is needed.

Reply:
i believe in most pages i visited, it was `No <item> to view.`